GWAS/PCA.ipynb

```diff
@@ -1177,7 +1188,8 @@
     "    unit = 4\n",
     "    n_col = 4\n",
     "    n_row = ceiling(k / n_col)\n",
-    "    plots = lapply(1:(k-1), function(i) plot_pcs(pca_final, paste0(\"PC\",i), paste0(\"PC\",i+1), dat$meta))\n",
+    "    maha_k = ${maha_k}\n",
+    "    plots = lapply(1:(maha_k-1), function(i) plot_pcs(pca_final, paste0(\"PC\",i), paste0(\"PC\",i+1), dat$meta))\n",
```

Comment:
I think the plots should not be maha_k - 1; it should be k - 1.

Reply:
If we want to plot just important PCs we need a parameter to pass this to the unction(I think can be maha_k?). It can not be k because reducing k would trigger an error in plotting scree plot step.

Reply:
I see, so you are reusing `maha_k` here ... it is not very good in terms of clarity. 

How about this: the scree plot should not be limited by K. Rather it hsould just plot all the PVE available in the data! It does not make sense to limit a scree plot. So can you change it so the scree plot does not use `k` and the actual PCA plot uses `k`?